oracle-plugin/src/e2e-test/java/io.cdap.plugin/OracleClient.java

```diff
@@ -0,0 +1,90 @@
+import java.sql.Statement;
+import java.util.TimeZone;
+
+/**
+ *  Oracle client.
+ */
+public class OracleClient {
+
+  private static Connection getOracleConnection() throws SQLException, ClassNotFoundException {
+    TimeZone timezone = TimeZone.getTimeZone("UTC");
+    TimeZone.setDefault(timezone);
+    Class.forName("oracle.jdbc.driver.OracleDriver");
+    String databaseName = PluginPropertyUtils.pluginProp("databaseName");
+    return DriverManager.getConnection("jdbc:oracle:thin:@//" + System.getenv("ORACLE_HOST")
+                                         + ":" + System.getenv("ORACLE_PORT") + "/" + databaseName,
+                                       System.getenv("ORACLE_USERNAME"), System.getenv("ORACLE_PASSWORD"));
+  }
+
+  public static int countRecord(String table, String schema) throws SQLException, ClassNotFoundException {
+    String countQuery = "SELECT COUNT(*) as total FROM " + schema + "." + table;
+    try (Connection connect = getOracleConnection(); Statement statement = connect.createStatement();
+         ResultSet rs = statement.executeQuery(countQuery)) {
+      int num = 0;
+      while (rs.next()) {
+        num = (rs.getInt(1));
+      }
+      return num;
+    }
+  }
+
+  public static void createSourceTable(String sourceTable, String schema) throws SQLException, ClassNotFoundException {
+    try (Connection connect = getOracleConnection(); Statement statement = connect.createStatement()) {
+      String createSourceTableQuery = "CREATE TABLE " + schema + "." + sourceTable +
+        "(ID number(38), LASTNAME varchar2(100))";
+      statement.executeUpdate(createSourceTableQuery);
+
+      // Truncate table to clean the data of last failure run.
+      String truncateSourceTableQuery = "TRUNCATE TABLE " + schema + "." + sourceTable;
+      statement.executeUpdate(truncateSourceTableQuery);
+
+      // Insert dummy data.
+      statement.executeUpdate("INSERT INTO " + schema + "." + sourceTable + " (ID, LASTNAME)" +
+                                " VALUES (1, 'Shelby')");
+      statement.executeUpdate("INSERT INTO " + schema + "." + sourceTable + " (ID, LASTNAME)" +
+                                " VALUES (2, 'Simpson')");
+    }
+  }
+
+  public static void createTargetTable(String targetTable, String schema) throws SQLException, ClassNotFoundException {
+    try (Connection connect = getOracleConnection(); Statement statement = connect.createStatement()) {
+      String createTargetTableQuery = "CREATE TABLE " + schema + "." + targetTable +
+        "(ID number(38), LASTNAME varchar2(100))";
+      statement.executeUpdate(createTargetTableQuery);
+    }
+  }
+
+  public static void deleteTables(String schema, String sourceTable, String targetTable)
+    throws SQLException, ClassNotFoundException {
+    try (Connection connect = getOracleConnection(); Statement statement = connect.createStatement()) {
+      String dropSourceTableQuery = "DROP TABLE " + schema + "." + sourceTable;
```

Comment:
Can we use String[] instead of sourceTable and targetTable and run the delete query in loop.

Reply:
Changed it to String[]